tests/Microsoft.Bot.Builder.TestBot/Controllers/BotController.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+
+namespace Microsoft.Bot.Builder.TestBot.Controllers
+{
+    [Route("bot")]
```

Comment:
This is going to cause confusion. :) 

Devs (myself included) are used to the "/api/messages" path, and are used to putting that into the emulator. Can you add a comment here saying, "Here's the URL to put into the emulator for local debugging" and show the example.

Reply:
or I could change it to "api/messages" ???
- and yes it confused me when I deployed to Azure and it didn't work until i fixed up the URL .... aaahhh